db/migrate/20220318153151_add_published_at_index_to_version.rb

```diff
@@ -0,0 +1,5 @@
+class AddPublishedAtIndexToVersion < ActiveRecord::Migration[5.2]
+  def change
+    add_index :versions, [:project_id, :published_at]
```

Comment:
I'm actually not sure if this index is worth it. The queries should use the existing index by `project_id` and then sort from those results, which means it should only be sorting a few hundred at the most.

Reply:
Not sure if it is worth it either. But if we add, needs to be done with `algorithm: concurrently` to avoid downtime